src/eq_schema/Group.js

```diff
@@ -1,15 +1,40 @@
+/* eslint-disable camelcase */
 const Block = require("./Block");
 const { getInnerHTML } = require("../utils/HTMLUtils");
+const { compact, isEmpty } = require("lodash");
 
 class Group {
   constructor(id, title, pages, ctx) {
     this.id = `group${id}`;
     this.title = getInnerHTML(title);
-    this.blocks = this.buildBlocks(this.title, pages, ctx);
+    this.blocks = this.buildBlocks(this.title, pages, ctx, id);
+
+    if (!isEmpty(ctx.routingGotos)) {
+      const skipConditions = this.buildSkipConditions(this.id, ctx);
+
+      if (!isEmpty(skipConditions)) {
+        this.skip_conditions = skipConditions;
+      }
+    }
+  }
+
+  buildSkipConditions(currentId, ctx) {
+    return compact(
+      ctx.routingGotos.map((rule, i) => {
+        if (rule.group === currentId) {
+          delete ctx.routingGotos[i];
```

Comment:
It's probably better to use a method such as `transform` rather than `map` to mutate  `ctx.routingGotos`.  Mutating an array/object in place can cause unintended side effects.